### ISSUE TITLE
Where to add grid creation functions?

### ISSUE THREAD

**Issue by jordancaraballo** [nasa-nccs-hpda/vhr-composite#10]
I have a couple of functions I created to streamline the creation of the grids etc. Is this the repo to include it into? If so, where should it go?

**Comment by cssprad1** [comment]
Yes, my original thought is to place it as a submodule, something like vhr-composite.grid. It could be a command-line application or a module invoked in python code (this is what we've been previously doing so I lean towards it).

**Comment by jordancaraballo** [comment]
Done
